src/Symfony/Component/Translation/Tests/Dumper/XliffFileDumperTest.php

```diff
@@ -38,4 +38,33 @@ public function testDump()
 
         unlink($tempDir.'/messages.en_US.xlf');
     }
+
+    public function testExtendedDumperCanSetCustomToolInfo()
+    {
+        $dumper = new XliffExtendedFileDumper();
+
+        $this->assertContains(
+            '<tool tool-id="symfony" tool-name="Test" tool-version="0.0"/>',
+            $dumper->formatTestMessages( array() )
+        );
+    }
+
 }
+
+/**
+ * Test subclass for configuring dumper via protected functions.
+ */
+class XliffExtendedFileDumper extends XliffFileDumper {
+    public function formatTestMessages( array $messages ){
```

Comment:
why do you have all theses spaces there?

@javiereguiluz is this valid? fabbot didn't get angry :smile: